Sources/Model/GraphQL/TransferMethodConfiguration.swift

```diff
@@ -162,22 +164,26 @@ struct TransferMethodConfigurationField: Codable {
     var transferMethodUIConfigurations: Connection<TransferMethodConfiguration>?
 }
 
+struct TransferMethodUpdateConfigurationField: Codable {
+    var transferMethodUpdateUIConfigurations: Connection<TransferMethodConfiguration>?
+}
+
 struct TransferMethodConfigurationKey: Codable {
     let countries: Connection<HyperwalletCountry>?
 }
 
 /// Representation of the transfer method configuration
 public struct TransferMethodConfiguration: Codable {
```

Comment:
update this to HyperwalletTransferMethodConfiguration -> any class/struct that is exposed to the public contains Hyperwallet prefix in the name

Reply:
Sure will update it.